src/dotnet/commands/dotnet-sln/LocalizableStrings.Designer.cs

```diff
@@ -0,0 +1,145 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
This file can be deleted.  We generate the .cs files for our .resx files on build.

Reply:
I removed it in 91ed48636.